editor/editor_run.cpp

```diff
@@ -178,7 +183,7 @@ Error EditorRun::run(const String &p_scene, const String &p_custom_args, const L
 
 	if (p_custom_args != "") {
 		Vector<String> cargs = p_custom_args.split(" ", false);
-		for (int i = 0; i < cargs.size(); i++) {
+		for (unsigned i = 0U; i < cargs.size(); i++) {
```

Comment:
Godot `Vector` (and all other container) size is signed int:
https://github.com/godotengine/godot/blob/9e34ba48556b71f1f672b5be47d12b0e3f679b86/core/vector.h#L87